Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/AnisotropyWrapper.java

```diff
@@ -146,23 +146,25 @@ public void run() {
 		final List<Subspace<BitType>> subspaces = HyperstackUtils.split3DSubspaces(
 			bitImgPlus).collect(toList());
 		matchOps(subspaces.get(0));
-		// TODO Does it make more sense to collect all the results we can (without
-		// cancelling) and then report errors at the end?
+		final List<Ellipsoid> ellipsoids = new ArrayList<>();
 		for (int i = 0; i < subspaces.size(); i++) {
 			statusService.showStatus("Anisotropy: sampling subspace #" + (i + 1));
 			final Subspace<BitType> subspace = subspaces.get(i);
-			if (!processSubspace(subspace)) {
+			final Ellipsoid ellipsoid = foo(subspace);
+			if (ellipsoid == null) {
```

Comment:
`foo` seems like a too general method name to me?

Reply:
Oops - good catch - not indeed a good name. Must've forgotten to change that.